tower_cli/api.py

```diff
@@ -156,8 +166,7 @@ def _make_request(self, method, url, args, kwargs):
                 'Right now it is: "%s".' % settings.host
             )
 
-    @property
-    def prefix(self):
+    def get_prefix(self, include_version=True):
```

Comment:
Up until now, every endpoint we've wanted to hit has been prefixed with a version, so I had to change this from a property to a function for generating `/api/o/` prefixes.